spec/high_voltage/configuration_spec.rb

```diff
@@ -48,7 +48,20 @@
     end
   end
 
-  describe '#action_caching_layout=' do
+  describe '#action_caching=false' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.